src/map/pc.c

```diff
@@ -1749,7 +1749,7 @@ int pc_calc_skilltree_normalize_job(struct map_session_data *sd)
 	sd->sktree.second = sd->sktree.third = 0;
 
 	// limit 1st class and above to novice job levels
-	if(skill_point < novice_skills) {
+	if(skill_point < novice_skills && (sd->class_&MAPID_BASEMASK) != MAPID_SUMMONER) {
```

Comment:
If possible, please later reorder/squash this into the commit that adds support for this job